applications/rubintv/secrets.yaml

```diff
@@ -14,13 +14,6 @@ redis-password:
     consumes those rapid-analysis put there.
   onepassword:
     encoded: true
-"aws-credentials.ini":
-  description: >-
-    S3 butler credentials to the Butler data store, formatted using
-    AWS syntax for use with boto.
-  copy:
-    application: obssys
-    key: "butler-credentials.ini"
```

Comment:
This removal might have been to run the `phalanx secrets sync` command right? I'm already fixing this on https://github.com/lsst-sqre/phalanx/pull/5188, please roll this back and I'll make sure to merge my PR ASAP 🙏.